spec/syntax.md

```diff
@@ -764,6 +766,11 @@ The _value_ of an _attribute_ can be either a _literal_ or a _variable_.
 
 Multiple _attributes_ are permitted in an _expression_ or _markup_.
 Each _attribute_ is separated by whitespace.
+Each _attribute_'s _identifier_ MUST be unique within the _expression_ or _markup_:
+an _expression_ or _markup_ with duplicate _attribute_ _identifiers_ is not valid.
+
+The order of _attributes_ is not significant.
```

Comment:
Let's not introduce this now. In this version, attributes are reserved. We didn't agree to make the order insignificant or require uniqueness, even if both become restrictions later.

Reply:
They're still reserved, but this makes attributes easier to deal with. The more different attributes and options are, the more custom code is required for dealing with this reserved syntax feature.

Reply:
The uniqueness requirement requires checking if the attribute is, in fact, unique. If one is using a map, perhaps that's easy. But if one is not, it's extra work that we just requirement.

I could maybe buy the ordering requirement, as this allows implementations to reorder attributes.

Reply:
I checked our three current implementations, and indeed none currently check attribute uniqueness, so this should be dropped for now.

```suggestion

The order of _attributes_ is not significant.
```